tests/src/tools/ReadyToRun.SuperIlc/ProcessRunner.cs

```diff
@@ -0,0 +1,220 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class ProcessInfo
+{
+    /// <summary>
+    /// 10 minutes should be plenty for a CPAOT / Crossgen compilation.
+    /// </summary>
+    public const int DefaultTimeout = 600 * 1000;
+
+    public string ProcessPath;
+    public string Arguments;
+    public bool UseShellExecute;
+    public string LogPath;
+    public int TimeoutMilliseconds = DefaultTimeout;
+    public int ExpectedExitCode;
+    public object Data;
```

Comment:
Can we strongly type this and use a meaningful name, ie, `FileName`?

Reply:
OK, will do, I was originally thinking it might also serve other purposes but I agree it's better not to introduce speculative code.